example/client/client.go

```diff
@@ -20,7 +20,7 @@ func ExtendedPostCmd(ctx context.Context, c *client.Client, data *types.Extended
 	defer cancel()
 
 	var outStr string
-	err := c.Query(queryCtx, "POST", api.NewURL().Path("extended"), data, &outStr)
+	err := c.Query(queryCtx, "POST", "1.0", api.NewURL().Path("extended"), data, &outStr)
```

Comment:
Can we use the `types.ExtendedEndpoint` const here?

Reply:
No, since it's from an internal package. This hardcoded value should get removed by #142.

Reply:
Wasn't it moved away from the internal package as part of https://github.com/canonical/microcluster/pull/155/commits/e7c64e38c8b4de6b9fd7cb902827e4005dc7af04?

Reply:
The type was, so external projects can set it in their `Server` structs after #142.

But all internally defined constants are in the [internal package](https://github.com/canonical/microcluster/blob/e7c64e38c8b4de6b9fd7cb902827e4005dc7af04/internal/rest/types/server.go)

Reply:
Thanks, clear now.